docs/reference/configuration/keywords.rst

```diff
@@ -677,43 +677,145 @@ FIELD
-applied. The available functions are listed below:
+.. _field-initialization:
+
+Field initialization
+^^^^^^^^^^^^^^^^^^^^
+
+Observe that ERT can *not* sample field variables internally, they must be
+supplied through another application - typically geo modelling software like
+RMS; so to use the FIELD datatype you must have a workflow external to ERT which
+can create/sample the fields. When you have established a workflow for
+generating these fields externally there are *two* ways to load them into ERT:
+`INIT_FILES` to load pregenerated initial fields or `FORWARD_INIT` to load as
+part of the forward model.
+
+.. _forward-init:
+
+Initialization with FORWARD_INIT
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When geomodelling is an integrated part of the forward model it is more
+attractive to let the forward model generate the parameter fields. To enable
+this we must pass the ``FORWARD_INIT:True`` when configuring the field, and also
+pass a name in the ``INIT_FILES:poro.grdecl`` for the file which should be
+generated by the forward model component.
+
+Observe that there are two important differences to the ``INIT_FILES:``
+attribute when it used as *the way* to initialize fields, and when it is used in
+combination with ``FORWARD_INIT:True``. When ``INIT_FILES:`` is used alone the
+filename given should contain a ``%d`` which will be replaced with realization
+number, when used with ``FORWARD_INIT:True`` that is not necessary. Furthermore
+in the ``FORWARD_INIT:True`` case *the path is interpreted relative to the
+runpath folder*, whereas in the other case the path is interpreted relative to
+the location of the main ERT configuration file.
+
+When using ``FORWARD_INIT:True`` together with an update algorithm in ERT the
+field generated by the geo modelling software should only be used in the first
+iteration (prior), in the subsequent iterations the forward model should use the
+field as it comes out from ERT. The typical way to achieve this is:
+
+1. The forward model component outputs to a temporary file ``tmp_poro.grdecl``.
+2. In the first iteration ERT will *not* output a file ``poro.grdecl``, but in
+   the second and subsequent iterations a ``poro.grdecl`` file will be created
+   by ERT - this is at the core of the ``FORWARD_INIT:True`` functionality.
+3. In the forward model there should be a job ``CAREFUL_COPY_FILE`` which will copy
+   ``tmp_poro.grdecl`` *only if* ``poro.grdecl`` does not already exist. The
+   rest of the forward model components should use ``poro.grdecl``.
+
+note
+  With regards to behavior relative to the values in storage;
+  What is really happening is that if ERT has values, those will be dumped
+  to the runpath, and if not, it will read those from the runpath after the
+  forward model finishes. However, if you change your runpath and "case" in
+  the config file, but not your storage case, you will end up with the same
+  parameter values but different RMS seed.
```

Comment:
Not sure I understand this section?

Reply:
I have not changed this sections and want to leave as is for now. Can update later.